contributed_definitions/nyaml/NXactuator.yaml

```diff
@@ -0,0 +1,106 @@
+category: base
+doc: |
+  An actuator used to control an external condition.
+  
+  The condition itself is described in :ref:`NXenvironment`.
+type: group
+NXactuator(NXobject):
+  model:
+    doc: |
+      Actuator identification code/model number
+  name:
+    doc: |
+      Name of the actuator
+  short_name:
+    doc: |
+      Short name of actuator used e.g. on monitor display program
+  attached_to:
+    doc: |
+      where actuator is attached to
+  actuation:
+    doc: |
+      name for quantity effected by actuation
+
+      Examples:
+      temperature | pH | magnetic_field | electric_field | current | conductivity | resistance | voltage |
+      pressure | flow | stress | strain | shear | surface_pressure
+  type:
+    doc: |
+      The type of hardware used for the actuation.
+
+      Examples (suggestions but not restrictions):
+      :Temperature:
+        laser | gas lamp | filament | resistive
+      :Pressure:
+        anvil cell
+      :Voltage:
+        potentiostat
+  setpoint(NX_FLOAT):
+    unit: NX_ANY
+    doc: |
+      Nominal setpoint
+      
+      Can be a scalar or a vector (of [n] actuations).
+    dimensions:
+      dim: [[1, n]]
+  setpoint_deriv1(NX_FLOAT):
+    doc: |
+      Nominal/average first derivative of setpoint
+      e.g. strain rate
+      - same dimensions as "setpoint" (may be a vector)
+    dimensions:
+      dim: [[1, ]]
+      dim_parameters:
+        ref: ['setpoint']
+  setpoint_deriv2(NX_FLOAT):
+    doc: |
+      Nominal/average second derivative of setpoint
+      - same dimensions as "setpoint" (may be a vector)
+    dimensions:
+      dim: [[1, ]]
+      dim_parameters:
+        ref: ['setpoint']
```

Comment:
What are these good for?
For the sensor data, the derivatives (and integrals) are used for PID control, but for the setpoints I don't see any obvious use case.

Reply:
You are right -> removed